src/go/rpk/pkg/cli/cmd/redpanda/config_test.go

```diff
@@ -239,7 +223,7 @@ schema_registry: {}
 			name: "set with loaded config",
 			cfgFile: `config_file: /etc/redpanda/redpanda.yaml
 redpanda:
-    data_directory: ""
+    data_directory: data/dir
```

Comment:
What's this from?

Reply:
I just changed the test content to show that we persist non-empty data_directory.

Reply:
i c, didn't realize these were inputs